app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -480,31 +322,15 @@ class TopicFragmentTest {
   fun testTopicFragment_clickOnReviewTab_configurationChange_showsSameTabAndItsContent() {
     launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
       testCoroutineDispatchers.runCurrent()
-      onView(
-        allOf(
-          withText(TopicTab.getTabForPosition(3).name),
-          isDescendantOfA(withId(R.id.topic_tabs_container))
-        )
-      ).perform(click())
+      clickTabAtPosition(position = 3)
       onView(isRoot()).perform(orientationLandscape())
       testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.topic_tabs_container)).check(
-        matches(
-          matchCurrentTabTitle(
-            TopicTab.getTabForPosition(3).name
-          )
-        )
-      )
-      onView(
-        atPositionOnView(R.id.revision_recycler_view, 0, R.id.subtopic_title)
-      ).check(
-        matches(
-          withText(
-            containsString(
-              "What is a Fraction?"
-            )
-          )
-        )
+      matchTabTitleAtPosition(position = 3)
+      matchStringOnListItem(
+        recyclerView = R.id.revision_recycler_view,
+        position = 0,
+        targetViewId = R.id.subtopic_title,
+        stringToMatch = "What is a Fraction?"
```

Comment:
Optional, but you could further specialize this for certain recycler views (e.g. new helpers that use this helper), such as:

```kotlin
verifySubtopicTitleIsPresent(subtopicPosition = 0, subtopicTitle = "What is a Fraction?")
```

Ditto for the other recycler views.

Reply:
as this is something optional, I will create a good-first-issue for this as the footfall of contributors is increasing a lot.

Reply:
Sounds good. Please mention the issue here once it exists (and add TODOs in code pointing to the issue where things need to change).

Reply:
#2208

Reply:
Left a reply on the issue, but otherwise lgtm.